README.md

```diff
@@ -106,6 +108,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
+| [Compare Europe Group](http://www.compareeuropegroup.com/) [:rocket:](http://www.compareeuropegroup.com/careers/) | Financial products comparision for several markets (ex: [ComparaJá](https://www.comparaja.pt)). | `Lisboa` | 
```

Comment:
Is it possible to shorten the company name?
Also, is it possible to shorten the description to fit in a single row to improve readability?
Thanks!

Reply:
I shorten the company name and was able to fit the description in a single line